### ISSUE TITLE
exclude some of FileTypes of binary files out of report 

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#414]
derived from the issue

- https://github.com/kazurayam/VisualInspectionOfExcelAndPDF/issues/13

**Comment by kazurayam** [comment]
The current version of the materialstore report used the version 5.1 of Bootstrap.

It uses the accordion of Bootstrap.

- https://mdbootstrap.com/docs/standard/components/accordion/

If you want to collapse a row, you can use the Bootstrap Collaps:

- https://mdbootstrap.com/docs/standard/components/collapse/

The switch widget can be rendered using the Bootstra  Switch

-

**Comment by kazurayam** [comment]
How to integrate these components together? Any example of javascript?


-> https://getbootstrap.com/docs/5.0/components/collapse/

**Comment by kazurayam** [comment]
I added

https://github.com/kazurayam/materialstore/blob/issue414/src/test/fixtures/issue%23414/page.html

in there

```
<div style="display: flex;">
        <div class="form-check form-switch" style="padding-right: 20px;">
            <input class="form-check-input" type="checkbox" id="flexSwitchCheckDefault1" checked>
            <label class="form-check-label" for="flexSwitchCheckDefault1">csv</label>
        </div>
...
```

```
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js"></script>
<script type="text/javascript">
    $('.form-switch input[type="checkbox"]').on('change', function() {
        console.log('Yaay, I was changed');
    });
</script>
```

In the console, when i toggle the swith I saw the on-event handler is invoked by jQuery:

<img width="832" alt="スクリーンショット 2023-05-07 10 48 36" src="https://user-images.githubusercontent.com/4467417/236653550-e598f77a-2387-4763-a8b8-e29eaa28ff40.png">

**Comment by kazurayam** [comment]
I tried to hide the rows when the FileType is switched off. I tried to implement it using the  Bootstrap Collapse to each div[class*="accordion-item"]. But I failed. The Bootstrap Accordion also uses the Bootstrap Collapse. Using the Collapse for 2 different subject corrided.

**Comment by kazurayam** [comment]
I should give up the idea of hiding the rows.
I am going to change the style of the rows of non-diffable FileTypes such as xlsx and pdf.
See the following screenshot.

**Comment by kazurayam** [comment]
See the source of sample HTML

https://github.com/kazurayam/materialstore/blob/768f9fefd82a44da2de3aa9daf2f220052a7c293/src/test/fixtures/issue%23414/page.html

**Comment by kazurayam** [comment]
done at the v0.16.4-SNAPSHOT

used the updated jar in the VisualInspectinOfExceAndPDF project; got a report like :

<img width="1552" alt="スクリーンショット 2023-05-08 9 33 56" src="https://user-images.githubusercontent.com/4467417/236710911-369124d7-01b6-40c8-9a31-aad35a6f8520.png">

<img width="1552" alt="スクリーンショット 2023-05-08 9 34 14" src="https://user-images.githubusercontent.com/4467417/236710914-9aed94b0-41d5-4087-b6a3-43fc79e2c196.png">

**Comment by kazurayam** [comment]
released as the v0.16.4
